spec/models/user_spec.rb

```diff
@@ -5,6 +5,10 @@
     it { should validate_presence_of :name }
     it { should validate_presence_of :email }
     it { should validate_uniqueness_of :email }
+    it { should allow_value('something@something.something').for(:email) }
+    it { should_not allow_value('something somthing@something.something').for(:email) }
+    it { should_not allow_value('something.something@').for(:email) }
+    it { should_not allow_value('something').for(:email) }
```

Comment:
Great addition to the model test validations to support the Sad paths you created!